test/compilable/test19463.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+
+$DMD -c -dip1008 -m${MODEL} -of${OUTPUT_BASE}${OBJ} -I${EXTRA_FILES} ${EXTRA_FILES}/${TEST_NAME}.d
+!(nm ${OUTPUT_BASE}${OBJ} | grep _d_newclass)
```

Comment:
@WalterBright As far as I understand this verifies that `_d_newclass` is **not** called, not that `_d_newThrowable` **is** called. I recommend implementing `_d_newThrowable` instead and verify it's getting called. This way a shell script is not required.